control-plane/pkg/reconciler/consumer/consumer.go

```diff
@@ -133,11 +133,11 @@ func (r *Reconciler) reconcileContractResource(ctx context.Context, c *kafkainte
 }
 
 func (r *Reconciler) reconcileContractEgress(ctx context.Context, c *kafkainternals.Consumer) (*contract.Egress, error) {
-	destination, err := r.Resolver.URIFromDestinationV1(ctx, c.Spec.Subscriber, c)
+	destination, err := r.Resolver.AddressableFromDestinationV1(ctx, c.Spec.Subscriber, c)
```

Comment:
```suggestion
	destinationAddr, err := r.Resolver.AddressableFromDestinationV1(ctx, c.Spec.Subscriber, c)
```

the same change in all other places in `consumer/consumer.go` and `trigger/trigger.go`